app/frontend/components/pages/receiveAda/addressItem.tsx

```diff
@@ -1,125 +1,127 @@
+      <div className={`expanded ${isExpanded ? '' : 'hide'}`}>
+        <div className="qr">
+          <img
+            src={new QRious({
+              value: address,
+              level: 'M',
+              size: 200,
+            }).toDataURL()}
+          />
         </div>
-        <div className="address-links blockexplorer-link">
-          <CopyOnClick value={address} elementClass="address-link copy">
-            <a className="copy-text">Copy Address</a>
-          </CopyOnClick>
-          <div style="margin-right:24px">
-            <span>View on </span>
-            {ADALITE_CONFIG.ADALITE_CARDANO_VERSION === 'byron' && (
-              <span>
-                <a
-                  className="address-link"
-                  href={`https://blockchair.com/cardano/address/${address}`}
-                  target="_blank"
-                  rel="noopener"
-                >
-                  Blockchair
-                </a>
-                <span> | </span>
-                <a
-                  className="address-link"
-                  href={`https://adascan.net/address/${address}`}
-                  target="_blank"
-                  rel="noopener"
-                >
-                  AdaScan
-                </a>
-              </span>
-            )}
-            {ADALITE_CONFIG.ADALITE_CARDANO_VERSION === 'shelley' && (
-              <span>
-                <a
-                  className="address-link"
-                  href={`https://cardanoscan.io/address/${address}`}
-                  target="_blank"
-                  rel="noopener"
-                >
-                  CardanoScan
-                </a>
-                {/* <span> | </span> */}
-                {/* <a
-                  className="address-link"
-                  href={`https://explorer.cardano.org/en/address?address=${address}`}
-                  target="_blank"
-                  rel="noopener"
-                >
-                  CardanoExplorer
-                </a> */}
-                <span> | </span>
-                <a
-                  className="address-link"
-                  href={`https://adaex.org/${address}`}
-                  target="_blank"
-                  rel="noopener"
-                >
-                  ADAex
-                </a>
-              </span>
-            )}
+        <div className="details">
+          {explorerLinks}
+          <div>
+            Derivation path: <span className="nowrap">{bip32path}</span>
+          </div>
+          <div className="desktop">
+            <CopyOnClick value={address} elementClass="address-link copy">
+              <a className="copy-text">Copy Address</a>
+            </CopyOnClick>
+          </div>
+          <div className="verification">
+            <AddressVerification address={address} />
           </div>
-          <a
-            className="address-link more"
-            onClick={() => this.openAddressDetail({address, bip32path}, this.state.onSmallDevice)}
```

Comment:
openAddressDetail is now completely unused

Reply:
also most likely the whole AddressDetailDialog class and closeAddressDetail

Reply:
I think the whole pop verify address modal logic is now unused, not sure if want to keep it or delete it all.

Reply:
Removed dead code.